include/oneapi/dpl/internal/dynamic_selection_impl/concurrent_queue.h

```diff
@@ -0,0 +1,71 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef _ONEDPL_CONCURRENT_QUEUE_H
+#define _ONEDPL_CONCURRENT_QUEUE_H
+
+#include <queue>
+#include <mutex>
+#include <deque>
```

Comment:
There is no need to include both `<queue>` and `<deque>`, is there?

Reply:
All includes a being reviewed to remove uncessary includes, remove cyclic includes and add missed includes.

Reply:
Concurrent_queueu header is removed.